### DISCUSSION TITLE
Leecher?

### DISCUSSION THREAD

**Discussion by 9Pigs** [c0re100/qBittorrent-Enhanced-Edition#243] [Q&A] [answered]
Pardon my ignorance -- but what identifies a leecher that qualifies to be blocked?

**Comment by InstaZDLL** [comment, marked as answer]
is someone who downloads a file and after that, he decided to not seed the file and when you delete the torrent from the BitTorrent client, the client will announce your share ratio to a peer from DHT, PEX, LSD or Trackers, the fact you will no longer have the file anymore, you're a leecher when you didn't reach the 1.00 of share ratio and you stop the seeding.
